docs/guides/events-list.rst

```diff
@@ -105,11 +105,14 @@ User events
 Relationship events
 ===================
 
-**create, <relationship>**
+**create, relationship**
```

Comment:
We can, I assumed we wouldn't want deprecated calls in the official docs though.